ufl/finiteelement/elementlist.py

```diff
@@ -200,6 +200,10 @@ def show_elements():
                   ("quadrilateral",))
 register_element2("BDMCF", 1, HDiv, "contravariant Piola", (1, None),
                   ("quadrilateral",))
+register_element2("SminusE", 1, HCurl, "covariant Piola", (1, None), cubes[1:3])
+register_element2("SminusF", 1, HDiv, "contravariant Piola", (1, None), cubes[1:2])
```

Comment:
Is there a reason that `SminusF` is not allowed on Hexahedra?

Reply:
After digging a bit, it looks like we have two attempts at this (from Cyrus + Justin) that have come in under different names and with slightly different implementations?  I think Cyrus put in SminusE/F and Justin put in SminusDiv/Curl.  The latter supports hexes in practice, but I don't see what the former shouldn't in theory (currently, it's disallowed in FIAT on hexes).

Reply:
If these are the same math object, perhaps we should merge them/make one an alias and wipe out the FI(n)AT implementation of the less complete one?